src/palette.st.css

```diff
@@ -61,6 +61,7 @@
 
 /* layout */
 :vars {
+  dividers: value(B50);
```

Comment:
There's also dividerColor here, and the difference between them is unclear to me.